filters/apiusagemonitoring/spec.go

```diff
@@ -73,6 +76,7 @@ func NewApiUsageMonitoring(
 		}
 	}
 	unknownPath := newPathInfo(
+		unknownPlaceholder,
 		unknownPlaceholder,
```

Comment:
Not completely clear to me what's the difference between when the `noTagPlaceholder` or the `unknownPlaceholder` is used.

Reply:
It's for giving `unknown` elements a semantic nice looking consistent value instead of empty string.